examples/app_engine/main.tf

```diff
@@ -14,23 +14,34 @@
  * limitations under the License.
  */
 
-locals {
-  credentials_file_path = "${path.module}/sa-key.json"
+provider "google" {
+  version = "~> 2.18.1"
 }
 
-/******************************************
-  Provider configuration
- *****************************************/
-provider "gsuite" {
-  credentials = file(local.credentials_file_path)
-  version     = "~> 0.1.12"
+provider "google-beta" {
+  version = "~> 2.18.1"
+}
+
+resource "random_string" "suffix" {
+  length  = 4
+  special = false
+  upper   = false
+}
+
+module "app-engine-project" {
+  source            = "../../"
+  name              = "appeng-${random_string.suffix.result}"
+  random_project_id = "true"
+  org_id            = var.org_id
+  folder_id         = var.folder_id
+  billing_account   = var.billing_account
+  activate_apis = [
+    "appengine.googleapis.com",
+  ]
 }
 
 module "app-engine" {
-  source           = "../../modules/app_engine"
-  location_id      = var.location_id
-  auth_domain      = var.auth_domain
```

Comment:
The `auth_domain` variable should be removed since it is not used.

Reply:
I havent used ```auth_domain``` in the tests since I did not know what example domain to use but it is still a valid input to the resource.

Reply:
Sorry, I just mean from the example. We aren't using it any more so it should be removed.

Reply:
Oh. On a second look, it has already been removed. Ignore me. :sweat_smile: